workshop/exercise-1/README.md

```diff
@@ -2,26 +2,26 @@
 
 Assume you already have a Kubernetes cluster, here are the steps to access your cluster:
 
-### Install IBM Cloud Container Service command line utilities
+## Install IBM Cloud Container Service command line utilities
 
 1. Install the IBM Cloud [command line interface](https://clis.ng.bluemix.net/ui/home.html).
 
-2. Log in to the IBM Cloud CLI with IBM API key:   
-   `bx login -u ibmcloudxx@us.ibm.com --apikey xxxx`      
+2. Log in to the IBM Cloud CLI:   
+   `bx login`      
 
 3. Install the IBM Cloud Container Service plug-in with `bx plugin install container-service -r Bluemix`.
 
 4. To verify that the plug-in is installed properly, run `bx plugin list`. The Container Service plug-in is displayed in the results as `container-service`.
 
-5. Initialize the Container Service plug-in and point the endpoint to your region, e.g. if your region is us-east:   
-   `bx cs region-set us-east`
+5. Initialize the Container Service plug-in and point the endpoint to your region:   
```

Comment:
Similar to above, there are prompts for region. Might be easier than if the customer doesn't know the region names off-hand.